packages/storybook/stories/ButtonV2.stories.tsx

```diff
@@ -47,7 +47,7 @@ export const Sizes: Story = {
             </Button>
           </span>
           <span key={size + '_iconOnly'}>
-            <Button {...props} size={size} icon={<ShareIcon />} Tag="span" />
+            <Button {...props} size={size} icon={<ShareIcon />} />
```

Comment:
Why is Tag span removed here?

Reply:
It is not really an accepted prop. It only accepts `a` or `button`.

Reply:
Lol, yeah. TS would go crazy until it gets the correct setup. We can maybe have it as an improvement in the future, part of V3 as you mentioned 👀